packages/millicast-sdk/src/utils/Diagnostics.js

```diff
@@ -0,0 +1,60 @@
+import { version } from '../../package.json'
+
+const MAX_STATS_HISTORY_SIZE = 5
+const diagnostics = {
+  version: '',
+  accountId: '',
+  streamName: '',
+  subscriberId: '',
+  timestamp: '',
+  userAgent: '',
+  connection: '',
+  stats: []
+}
+
+const setAccountId = (accountId) => { diagnostics.accountId = accountId }
+
+const setStreamNameAndSubscriberId = (streamName, subscriberId) => {
+  diagnostics.streamName = streamName
+  diagnostics.subscriberId = subscriberId
+}
+
+const setStreamViewId = (streamViewId) => { diagnostics.streamViewId = streamViewId }
+
+const setFeedId = (feedId) => { diagnostics.feedId = feedId }
+
+const setConnectionState = (connectionState) => { diagnostics.connection = connectionState }
+
+const setStats = (stats) => {
+  if (diagnostics.stats.length === MAX_STATS_HISTORY_SIZE) {
+    diagnostics.stats.shift()
+  }
+  diagnostics.stats.push(stats)
+}
+
+const getDiagnostics = async (statsCount = MAX_STATS_HISTORY_SIZE) => {
```

Comment:
Do we need async function here?
```suggestion
const getDiagnostics = (statsCount = MAX_STATS_HISTORY_SIZE) => {
```